fix for 'error: identifier "CudaLaunchKernel" is undefined' when building TensorFlow 1.14.0 with CUDA + MPI
see https://github.com/tensorflow/tensorflow/issues/30703
--- tensorflow-1.14.0/tensorflow/contrib/mpi_collectives/kernels/ring.cu.cc.orig	2019-09-21 09:13:22.101634483 +0200
+++ tensorflow-1.14.0/tensorflow/contrib/mpi_collectives/kernels/ring.cu.cc	2019-09-21 09:13:34.551977959 +0200
@@ -21,6 +21,7 @@
 
 #include "tensorflow/contrib/mpi_collectives/kernels/ring.h"
 #include "tensorflow/core/util/gpu_launch_config.h"
+#include "tensorflow/core/util/gpu_kernel_helper.h"
 
 namespace tensorflow {
 namespace contrib {
